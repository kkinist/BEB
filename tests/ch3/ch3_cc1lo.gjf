%chk=ch3.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 8: low-spin CCSD(T) for cation of ch3

1 1

C H 0
cc-pVTZ
****

