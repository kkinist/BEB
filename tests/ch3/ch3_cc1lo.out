 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-28025.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     28027.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch3.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 ----------------------------------------------
 BEB step 8: low-spin CCSD(T) for cation of ch3
 ----------------------------------------------
 Structure from the checkpoint file:  "ch3.chk"
 Charge =  1 Multiplicity = 1
 Redundant internal coordinates found in file.
 C,0,0.,-0.0000000011,0.
 H,0,0.000000002,-1.0827517802,0.
 H,0,-0.9376905476,0.5413758867,0.
 H,0,0.9376905457,0.5413758901,0.
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000   -1.082752    0.000000
      3          1           0       -0.937691    0.541376    0.000000
      4          1           0        0.937691    0.541376    0.000000
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4
     1  C    0.000000
     2  H    1.082752   0.000000
     3  H    1.082752   1.875381   0.000000
     4  H    1.082752   1.875381   1.875381   0.000000
 Stoichiometry    CH3(1+)
 Framework group  D3H[O(C),3C2(H)]
 Deg. of freedom     1
 Full point group                 D3H     NOp  12
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.000000
      2          1           0        0.000000    1.082752    0.000000
      3          1           0       -0.937691   -0.541376    0.000000
      4          1           0        0.937691   -0.541376    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    285.1563071    285.1563071    142.5781536
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.8236000000D+04  0.5419783203D-03
      0.1235000000D+04  0.4192873817D-02
      0.2808000000D+03  0.2152216205D-01
      0.7927000000D+02  0.8353432195D-01
      0.2559000000D+02  0.2395828457D+00
      0.8997000000D+01  0.4428528419D+00
      0.3319000000D+01  0.3517995618D+00
 S   6 1.00       0.000000000000
      0.2808000000D+03 -0.5949224937D-04
      0.7927000000D+02 -0.1148158310D-02
      0.2559000000D+02 -0.1001913745D-01
      0.8997000000D+01 -0.6121949230D-01
      0.3319000000D+01 -0.1732698541D+00
      0.3643000000D+00  0.1072915192D+01
 S   1 1.00       0.000000000000
      0.9059000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1285000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1871000000D+02  0.3942638716D-01
      0.4133000000D+01  0.2440889849D+00
      0.1200000000D+01  0.8154920089D+00
 P   1 1.00       0.000000000000
      0.3827000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1209000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1097000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.3180000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.7610000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      4 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    35 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     8 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    22 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    15 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    30 symmetry adapted basis functions of A1  symmetry.
 There are     8 symmetry adapted basis functions of A2  symmetry.
 There are    20 symmetry adapted basis functions of B1  symmetry.
 There are    14 symmetry adapted basis functions of B2  symmetry.
    72 basis functions,   103 primitive gaussians,    80 cartesian basis functions
     4 alpha electrons        4 beta electrons
       nuclear repulsion energy         9.6437168407 Hartrees.
 NAtoms=    4 NActive=    4 NUniq=    2 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    72 RedAO= T EigKep=  2.77D-03  NBF=    30     8    20    14
 NBsUse=    72 1.00D-06 EigRej= -1.00D+00 NBFU=    30     8    20    14
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (?A) (?A) (?B) (?B)
       Virtual   (?C) (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?B) (?B)
                 (?A) (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D)
                 (?B) (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?C) (?A) (?D) (?D)
                 (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?C)
                 (?A) (?B) (?B) (?D) (?D) (?C) (?A) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Keep R1 ints in memory in symmetry-blocked form, NReq=5456056.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RHF) =  -39.2477364956     A.U. after    8 cycles
            NFock=  8  Conv=0.16D-08     -V/T= 2.0007
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 8.24D+03 ExpMxC= 2.81D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    72
 NBasis=    72 NAE=     4 NBE=     4 NFC=     1 NFV=     0
 NROrb=     71 NOA=     3 NOB=     3 NVA=    68 NVB=    68

 **** Warning!!: The largest alpha MO coefficient is  0.11843536D+02

 Semi-Direct transformation.
 ModeAB=           4 MOrb=             3 LenV=    1048274434
 LASXX=       134806 LTotXX=      134806 LenRXX=      277006
 LTotAB=      142200 MaxLAS=      513756 LenRXY=           0
 NonZer=      411812 LenScr=     1234944 LnRSAI=      513756
 LnScr1=     1541120 LExtra=           0 Total=      3566826
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=0 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.2547806572D-02 E2=     -0.8496422076D-02
     alpha-beta  T2 =       0.3259458973D-01 E2=     -0.1147996563D+00
     beta-beta   T2 =       0.2547806572D-02 E2=     -0.8496422076D-02
 ANorm=    0.1018670802D+01
 E2 =    -0.1317925004D+00 EUMP2 =    -0.39379528996052D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=5403492.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 MP4(R+Q)=  0.22425888D-02
 E3=       -0.18452928D-01        EUMP3=      -0.39397981924D+02
 E4(DQ)=   -0.23610413D-02        UMP4(DQ)=   -0.39400342966D+02
 E4(SDQ)=  -0.27118099D-02        UMP4(SDQ)=  -0.39400693734D+02
 DE(Corr)= -0.14798321     E(Corr)=     -39.395719705    
 NORM(A)=   0.10258491D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15238733     E(CORR)=     -39.400123827     Delta=-4.40D-03
 NORM(A)=   0.10276779D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15338909     E(CORR)=     -39.401125585     Delta=-1.00D-03
 NORM(A)=   0.10279340D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15354579     E(CORR)=     -39.401282289     Delta=-1.57D-04
 NORM(A)=   0.10279594D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15355362     E(CORR)=     -39.401290111     Delta=-7.82D-06
 NORM(A)=   0.10279598D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15355346     E(CORR)=     -39.401289954     Delta= 1.57D-07
 NORM(A)=   0.10279599D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15355341     E(CORR)=     -39.401289910     Delta= 4.37D-08
 NORM(A)=   0.10279601D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        12
 NAB=     6 NAA=     0 NBB=     0.
 DE(Corr)= -0.15355345     E(CORR)=     -39.401289944     Delta=-3.40D-08
 NORM(A)=   0.10279601D+01
 Largest amplitude= 5.52D-02
 Time for triples=        0.86 seconds.
 T4(CCSD)= -0.30483356D-02
 T5(CCSD)=  0.51270029D-04
 CCSD(T)= -0.39404287010D+02
 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (?A) (?A) (?B) (?B)
       Virtual   (?C) (?A) (?B) (?B) (?B) (?B) (?C) (?A) (?A) (?B)
                 (?B) (?D) (?D) (?A) (?A) (?B) (?B) (?C) (?D) (?D)
                 (?B) (?B) (?B) (?B) (?A) (?C) (?A) (?B) (?B) (?B)
                 (?B) (?D) (?D) (?A) (?C) (?A) (?B) (?B) (?D) (?D)
                 (?C) (?A) (?B) (?B) (?D) (?D) (?B) (?B) (?A) (?A)
                 (?C) (?B) (?B) (?D) (?D) (?C) (?B) (?B) (?A) (?B)
                 (?B) (?D) (?D) (?A) (?B) (?B) (?A) (?A)
 Unable to determine electronic state:  an orbital has unidentified symmetry.
 Alpha  occ. eigenvalues --  -11.68145  -1.27963  -0.95338  -0.95338
 Alpha virt. eigenvalues --   -0.28344  -0.08412  -0.01684  -0.01684   0.10126
 Alpha virt. eigenvalues --    0.10126   0.19479   0.20422   0.34348   0.35781
 Alpha virt. eigenvalues --    0.35781   0.42380   0.42380   0.72333   0.75610
 Alpha virt. eigenvalues --    0.77693   0.77693   0.85231   0.95286   0.95286
 Alpha virt. eigenvalues --    1.13734   1.13734   1.23321   1.23321   1.47789
 Alpha virt. eigenvalues --    1.99934   2.03501   2.18785   2.18785   2.46732
 Alpha virt. eigenvalues --    2.46732   2.53357   2.53357   2.80071   2.80973
 Alpha virt. eigenvalues --    2.84147   2.85516   2.85516   2.90008   2.90008
 Alpha virt. eigenvalues --    3.15053   3.18368   3.33390   3.33390   3.56402
 Alpha virt. eigenvalues --    3.56402   3.59447   3.59447   3.83479   3.86742
 Alpha virt. eigenvalues --    3.87277   3.93189   3.93189   4.04874   4.04874
 Alpha virt. eigenvalues --    4.62686   4.81185   4.81185   4.81928   4.86267
 Alpha virt. eigenvalues --    4.86267   4.91046   4.91046   5.53470   5.65074
 Alpha virt. eigenvalues --    5.65074   6.17559  12.83058
          Condensed to atoms (all electrons):
               1          2          3          4
     1  C    4.567391   0.389681   0.389681   0.389681
     2  H    0.389681   0.389512  -0.012335  -0.012335
     3  H    0.389681  -0.012335   0.389512  -0.012335
     4  H    0.389681  -0.012335  -0.012335   0.389512
 Mulliken charges:
               1
     1  C    0.263567
     2  H    0.245478
     3  H    0.245478
     4  H    0.245478
 Sum of Mulliken charges =   1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    1.000000
 Electronic spatial extent (au):  <R**2>=             22.3282
 Charge=              1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -4.0819   YY=             -4.0819   ZZ=             -4.9753
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              0.2978   YY=              0.2978   ZZ=             -0.5956
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              1.7255  ZZZ=              0.0000  XYY=              0.0000
  XXY=             -1.7255  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -7.7739 YYYY=             -7.7739 ZZZZ=             -3.3552 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -2.5913 XXZZ=             -2.4620 YYZZ=             -2.4620
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 9.643716840664D+00 E-N=-1.072657148709D+02  KE= 3.921848204342D+01
 Symmetry A1   KE= 3.702413127897D+01
 Symmetry A2   KE= 2.185175981265D-32
 Symmetry B1   KE= 2.194350764444D+00
 Symmetry B2   KE= 7.573738758393D-32
 1\1\GINC-GAMBA\SP\RCCSD(T)-FC\Gen\C1H3(1+)\IRIKURA\17-May-2017\0\\# CC
 SD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 8: low-spin 
 CCSD(T) for cation of ch3\\1,1\C,0,0.,-0.0000000011,0.\H,0,0.000000002
 ,-1.0827517802,0.\H,0,-0.9376905476,0.5413758867,0.\H,0,0.9376905457,0
 .5413758901,0.\\Version=ES64L-G09RevD.01\HF=-39.2477365\MP2=-39.379529
 \MP3=-39.3979819\MP4D=-39.4025856\MP4DQ=-39.400343\MP4SDQ=-39.4006937\
 CCSD=-39.4012899\CCSD(T)=-39.404287\RMSD=1.636e-09\PG=D03H [O(C1),3C2(
 H1)]\\@


 THERE IS NO SCIENCE WITHOUT FANCY, NOR ART WITHOUT FACTS.
     -- VLADIMIR NABAKOV
 Job cpu time:       0 days  0 hours  0 minutes 21.5 seconds.
 File lengths (MBytes):  RWF=     35 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:00:17 2017.
